Packs/Carbon_Black_Enterprise_Response/IncidentTypes/CarbonBlackEDR.json

```diff
@@ -0,0 +1,42 @@
+[
+	{
+		"id": "Carbon Black EDR",
+		"version": 3,
+		"modified": "2021-06-15T15:59:53.335159+03:00",
+		"sortValues": null,
+		"packID": "",
```

Comment:
i think some of the fields in this file, e.g packid are redundant. you can run format on this specific file.